.travis.yml

```diff
@@ -3,15 +3,17 @@ install:
   - pip install tox
   - python3 setup.py egg_info
   - pip install -r aiosmtpd.egg-info/requires.txt
-matrix:
+jobs:
   include:
     - python: "3.6"
       env: INTERP=py36
     - python: "3.7"
       env: INTERP=py37
     - python: "3.8"
       env: INTERP=py38
-    - python: "pypy3"
+    - python: "3.9"
+      env: INTERP=py39
+    - python: "pypy3.6-7.3.1"
```

Comment:
This is the latest PyPy3, and it had implemented the `bpo-27931` fix.

Don't know why `pypy3` is not an alias to this newest version, though... But all's well that ends well, and now it successfully builds for all 5 Pythons.